errors/errors.go

```diff
@@ -1,57 +1,19 @@
 package errors
 
-type SafeNotFound struct {
-}
+import (
```

Comment:
I explicitly prefer to keep it this way which makes future imports only change a single line of diff.
BTW: There's merely a single shorthand `import` statement in this project (which should adapt to the import-list style imo).